<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="DESKTOP-ICBR68V" name="com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense" tests="6" failures="0" timestamp="29 Jan 2018 13:28:47 GMT" time="109.696" errors="1">
  <testcase name="verifyEmployeeNameOnJobRoleAndGoBackToEmployment" time="0.258" classname="com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense"/>
  <testcase name="searchCreatedEmployee" time="53.840" classname="com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense"/>
  <testcase name="clickOnSearchedEmployee" time="11.901" classname="com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense"/>
  <testcase name="verifyJobTitleOfCreatedEmployee" time="5.443" classname="com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense"/>
  <testcase name="verifyRewardTabAndClickedRewardTab" time="38.254" classname="com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense">
    <error type="org.openqa.selenium.StaleElementReferenceException" message="Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 30.05 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &#039;3.0.1&#039;, revision: &#039;1969d75&#039;, time: &#039;2016-10-18 09:49:13 -0700&#039;
System info: host: &#039;DESKTOP-ICBR68V&#039;, ip: &#039;192.168.1.131&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_151&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 3dc93771-cb0c-4fd7-bd44-dd15521626ed">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked upCommand duration or timeout: 30.05 secondsFor documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.htmlBuild info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'System info: host: 'DESKTOP-ICBR68V', ip: '192.168.1.131', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'Driver info: org.openqa.selenium.firefox.FirefoxDriverCapabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]Session ID: 3dc93771-cb0c-4fd7-bd44-dd15521626edat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)at java.lang.reflect.Constructor.newInstance(Unknown Source)at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:635)at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:274)at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)at com.sun.proxy.$Proxy7.click(Unknown Source)at com.test.xcdhr.Core_Sence_Check.uiActions.JoinerFirstPage.clickRewardTab(JoinerFirstPage.java:450)at com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.verifyRewardTabAndClickedRewardTab(TC001_VerifyJoiner_WithLicense.java:99)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked upFor documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.htmlBuild info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'System info: host: 'DESKTOP-ICBR68V', ip: '192.168.1.131', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'Driver info: driver.version: unknownat <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web-element-cache.js:9528)at <anonymous class>.Utils.getElementAt(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10202)at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10714)at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12688)at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12705)at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:720)]]>
    </error>
  </testcase> <!-- verifyRewardTabAndClickedRewardTab -->
  <testcase name="verifyAnnualSalaryOfCreatedCompensation" time="0.000" classname="com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense">
    <skipped/>
  </testcase> <!-- verifyAnnualSalaryOfCreatedCompensation -->
</testsuite> <!-- com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense -->
